proto/p4/p4runtime.proto

```diff
@@ -356,6 +357,29 @@ message CloneSessionEntry {
   uint64 packet_length_bytes = 4;
 }
 
+//------------------------------------------------------------------------------
+// For writing and reading values in a parser value set. It is OK for the value
+// set to have no entries, as a result of a write. A state transition matching
+// on an empty value set will never be taken. Values contained in a value set
+// must be unique.
+//
+// For Write Requests:
+//    INSERT will add the given values in the repeated field to the existing
```

Comment:
To be consistent with rest of P4Runtime semantics ... suggest re-aligning this with ValueSet being the P4-entity in this case. So what we insert/modify is the entire value-set, and delete should only require its id.

Reply:
Done.